manager/state/raft/membership/cluster.go

```diff
@@ -180,7 +181,7 @@ func (c *Cluster) ReplaceMemberConnection(id uint64, oldConn *Member, newConn *M
 		return ErrIDNotFound
 	}
 
-	if oldConn.Conn != oldMember.Conn {
+	if !force && oldConn.Conn != oldMember.Conn {
```

Comment:
force was added in this pr, so it's somehow related to addr :) what the point of force argument?